chromium_src/chrome/browser/flag_descriptions.cc

```diff
@@ -62,6 +62,11 @@ const char kBraveSuperReferralDescription[] =
 const char kBraveEphemeralStorageName[] = "Enable Ephemeral Storage";
 const char kBraveEphemeralStorageDescription[] =
     "Use ephemeral storage for third-party frames";
+const char kBraveEphemeralStorageKeepAliveName[] =
+    "Ephermal Storage Keep Alive";
+const char kBraveEphemeralStorageKeepAliveDescription[] =
+    "Keep ephermal storage partitions alive for 30 seconds after last tab "
```

Comment:
> for 30 seconds

this will be bogus if ```kTimeoutTcpConnectAttemptMax``` value will change at some point.
I think it's safer to use our own feature parameter so we can have some stability/flexibility if chromium change/remove this value.